modules/menu/menu.c

```diff
@@ -402,6 +402,49 @@ static int cmd_ua_next(struct re_printf *pf, void *unused)
 	return err;
 }
 
+static int cmd_ua_find(struct re_printf *pf, void *arg)
+{
+	const struct cmd_arg *carg = arg;
+	struct ua *ua = NULL;
+	int err;
+
+	(void)pf;
+
+	if (str_isset(carg->prm)) {
+
+		mbuf_rewind(menu.dialbuf);
+		(void)mbuf_write_str(menu.dialbuf, carg->prm);
+
+		ua = uag_find_aor(carg->prm);
+	}
+	else if (menu.dialbuf->end > 0) {
+
```

Comment:
is this block needed ?

the logic is used to repeat the last aor string, it set. it is used in e.g. the dial command.
not sure that it is needed also here ?